app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+require './lib/user_helper.rb'
+
+class User < ApplicationRecord
+  include UserHelper
+
+  has_many :posts
+  has_many :comments
+  validates :user_name, uniqueness: true, length: { in: 6..15 }
+  validate :username_cant_be_nil
```

Comment:
- [ ] I strongly recommend not to use this method because you can use the presence validation

Reply:
Thank you, done